src/jaxsim/api/ode.py

```diff
@@ -272,18 +270,15 @@ def system_acceleration(
     )
 
     # - Joint accelerations: s̈ ∈ ℝⁿ
-    # - Base inertial-fixed acceleration: W_v̇_WB = (W_p̈_B, W_ω̇_B) ∈ ℝ⁶
-    with (
-        data.switch_velocity_representation(velocity_representation=VelRepr.Inertial),
-        references.switch_velocity_representation(VelRepr.Inertial),
-    ):
-        W_v̇_WB, s̈ = js.model.forward_dynamics_aba(
-            model=model,
-            data=data,
-            joint_forces=references.joint_force_references(),
-            link_forces=references.link_forces(),
-        )
-    return W_v̇_WB, s̈
+    # - Base acceleration: O_v̇_WB = (O_p̈_B, O_ω̇_B) ∈ ℝ⁶
```

Comment:
This is true only in mixed representation :)

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/223/commits/a09bc0da027d89f0b1f3c6e5b72e9df55c19f41c